### PR TITLE
HDDS-2734. start/RunningViaDocker.md translated to Chinese

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

doc translation

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2734

## How was this patch tested?

hugo server -D

### CODE REVIEW

**Comment by iamabug** [comment]
@cxorm @xiaoyuyao Could you help review this ? Thanks.

File: hadoop-hdds/docs/content/start/RunningViaDocker.zh.md [hunk 1]
```diff
@@ -21,40 +21,31 @@ weight: 23
...
 docker-compose up -d
 {{< /highlight >}}
 
-To verify that ozone is working as expected, let us log into a data node and
-run _freon_, the load generator for Ozone. The ```exec datanode bash``` command
-will open a bash shell on the datanode.
-
-The `ozone freon` command is executed within the datanode container. You can quit freon via CTRL-C any time. The
-```rk``` profile instructs freon to generate random keys.
+为了验证 Ozone 正常运行，我们可以登录到 Datanode 并运行 Ozone 的负载生成命令 _freon_。 ```exec datanode bash``` 命令会在 Datanode 上启动一个 bash，`ozone freon` 命令在 Datanode 所在的容器内执行，你随时可以通过  CTRL-C 退出 freon，```rk``` 指令会让 freon 生成随机的键。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/start/RunningViaDocker.zh.md]: 负载生成命令=>负载生成工具

Comment (xiaoyuyao) [reply hadoop-hdds/docs/content/start/RunningViaDocker.zh.md]: ```rk``` 指令=》命令行选项```rk```

File: hadoop-hdds/docs/content/start/RunningViaDocker.zh.md [hunk 2]
```diff
@@ -21,40 +21,31 @@ weight: 23
...
 
 {{< highlight bash >}}
 docker-compose exec datanode bash
 ozone freon rk
 {{< /highlight >}}
 
-You can check out the **OzoneManager UI** at http://localhost:9874/ to see the
-activity generated by freon.
-While you are there, please don't forget to check out the ozone configuration explorer.
+你可以访问 http://localhost:9874/ 在 **OzoneManager UI** 中查看 freon 的负载生成情况，同时也可以浏览 ozone 的配置。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/start/RunningViaDocker.zh.md]: 你可以访问 http://localhost:9874/ 在 **OzoneManager UI**中查看=》
你可以通过 http://localhost:9874/ 访问 **OzoneManager UI**来查看服务器端处理freon负载的情况，以及浏览ozone的配置。

**Pull request merged**